src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String WHITE = "white";

```

Comment:
WHITE for white doesn't add any value. first of all, let's call it by its role - DEFAULT_COLOR. Also, let's tie this to an enum, which was suggested in the previous comment (use .name())